Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ruby:2.6.3
+
+WORKDIR /workspace
+
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg -o /root/yarn-pubkey.gpg && apt-key add /root/yarn-pubkey.gpg
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list
+RUN curl -sL https://deb.nodesource.com/setup_12.x | /bin/bash -
+RUN apt-get update && apt-get install -y --no-install-recommends nodejs yarn
+
+RUN apt-get update && \
+  apt-get install -qq -y --no-install-recommends && \
+  rm -rf /var/lib/apt/lists/*
+
+
+COPY Gemfile ./
+COPY Gemfile.lock ./
+
+RUN apt install shared-mime-info
+RUN gem install mimemagic -v '0.3.10' --source 'https://rubygems.org/'
+RUN gem install rails bundler
+RUN bundle install
+
+COPY . ./
+RUN chmod +x /workspace/bin/rails
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | /bin/bash -
+
+
```

Comment:
@Mubasher112 some env variables are needed to be exported, for example:
`RECAPTCHA_SECRET_KEY=6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe RECAPTCHA_SITE_KEY=6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI`
can you also add a command that would add the env variables? otherwise sign up and login pages could not be accessed

Reply:
@Mubasher112 how about this one? did you saw this change request?